packages/next-app/src/app/stableswapsimulator/(components)/PoolParametersForm.tsx

```diff
@@ -1,13 +1,76 @@
 "use client";
 
-import { useStableSwap } from "#/contexts/StableSwapContext";
+import { useEffect, useState } from "react";
+
+import Button from "#/components/Button";
+import { Input } from "#/components/Input";
+import { AnalysisData, useStableSwap } from "#/contexts/StableSwapContext";
 
 export default function PoolParametersForm() {
-  const { initialData } = useStableSwap();
+  const { initialData, setInitialData } = useStableSwap();
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [formData, setFormData] = useState(initialData);
```

Comment:
do we even need this state here? Couldn't you just push everything to your context?